src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -72,15 +97,22 @@ public LocalDate addWeeks(LocalDate localDate, Integer numberOfWeeks) {
      *                  if `someDate` is today;
      */
     public String beforeOrAfter(LocalDate someDate) {
-        return someDate + "is today";
+        LocalDate now = LocalDate.now();
+        if (someDate.isAfter(now)) {
+            return someDate.toString() + " is after " + LocalDate.now();
+        } else if (someDate.equals(now)) {
```

Comment:
do we need `else` after `return` statement?

Reply:
I thought it's more readable in this way.